docs/getting_started.rst

```diff
@@ -399,12 +399,16 @@ Or to check the formatting without writing changes, run:
   yarn run lint-frontend
   yarn run fmt-backend:check
 
-To have code automatically formatted and checked for linting upon commit, you may use `pre-commit <http://pre-commit.com/>`__ this can help ensure clean, consistent code, and will prevent automated build checks due to linting errors. The pip package should already be installed from the Python dev dependency installation, but you need to install the git hooks using this command:
+`pre-commit <http://pre-commit.com/>`__ is used to apply a full set of checks and formatting automatically each time that ``git commit`` runs. If there are errors, the Git commit is aborted and you are asked to fix the error and run ``git commit`` again.
```

Comment:
```suggestion
`pre-commit <http://pre-commit.com/>`__ is used to check and format code added to your git staging area each time that ``git commit`` runs. If there are errors, the commit is aborted and you are asked to fix the error and run ``git commit`` again.
```

Reply:
I would really miss the "full set" detail, i.e. that pre-commit aligns with CI linting. I've had a lot of frustration with non-aligned local linting in this and my own repositories -- maybe other people won't read "full set" with the same ideas, but that's the idea.

Reply:
Ah, I interpreted "full set" to refer to the list of files.